spec.html

```diff
@@ -18284,7 +18284,7 @@ <h1>Static Semantics: Early Errors</h1>
         <emu-grammar>PropertyDefinition : CoverInitializedName</emu-grammar>
         <ul>
           <li>
-            Always throw a Syntax Error if code matches this production.
+            Always throw a Syntax Error if code is matched by this production.
```

Comment:
```suggestion
           It is a Syntax Error if this production participates in a parse.
```

Reply:
Alternative:
```suggestion
            It is a Syntax Error if any source text is matched by this production.
```

Reply:
Either WFM.

Reply:
I went with @gibson042's suggestion.

Or, we could put it in the active voice:
 "It is a Syntax Error if this production matches any source text."